src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -769,6 +769,197 @@ function Test-ApplicationGatewayCRUDRewriteRuleSet
 	}
 }
 
+function Test-ApplicationGatewayCRUDRewriteRuleSetWithConditions
+{
+	param
+	(
+		$basedir = "./"
```

Comment:
You will need to get the actual base dir from the current AppDomain (e.g. ```AppDomain.CurrentDomain.BesDirectory```)

Reply:
@markcowl the test is always invoked with BaseDirectory, see `ApplicationGatewayTests.cs`
```csharp
TestRunner.RunTestScript(string.Format("Test-ApplicationGatewayCRUDRewriteRuleSetWithConditions -baseDir '{0}'", AppDomain.CurrentDomain.BaseDirectory));
```